Tools/scripts/verify-ensurepip-wheels.py

```diff
@@ -0,0 +1,78 @@
+"""Compares checksums for wheels in :mod:`ensurepip` against the Cheeseshop."""
```

Comment:
Warehouse seems to be the new name https://github.com/python/cpython/pull/31885#issuecomment-1147849525, and docstring are usually imperative (others in `Tools/scripts` too)

```suggestion
"""Compare checksums for wheels in :mod:`ensurepip` against the Warehouse."""
```